app/services/LocationService.js

```diff
@@ -449,4 +453,70 @@ export default class LocationServices {
       // nav.navigate('LocationTrackingScreen', {});
     });
   }
+
+  static getCrossedPaths() {
```

Comment:
this returns a boolean and not the actual crossed paths, so perhaps `getHasCrossedPaths`